test/SILOptimizer/sr6709.swift

```diff
@@ -0,0 +1,35 @@
+// Make sure project_box gets assigned the correct lexical scope when we create it.
+// RUN: %target-swift-frontend -primary-file %s -Onone -emit-sil -Xllvm -sil-print-after=capture-promotion -Xllvm \
+// RUN:   -sil-print-debuginfo -o /dev/null 2>&1 | %FileCheck %s
+
+// CHECK: sil hidden @_T04null19captureStackPromoteSiycyF : $@convention(thin) () -> @owned @callee_guaranteed () -> Int {
+// CHECK: bb0:
+// CHECK:   %0 = alloc_box ${ var Int }, var, name "x", loc {{.*}}:32:7, scope 3 // users: %19, %7, %1
+// CHECK:   %1 = project_box %0 : ${ var Int }, 0, loc {{.*}}:32:7, scope 3 // users: %9, %6
```

Comment:
Leaving the comments in the CHECK lines is probably just asking for trouble. I would remove them.

Reply:
I'll update this :)